app-with-redux-toolkit/src/redux/services.ts

```diff
@@ -0,0 +1,27 @@
+let accountBalance = 0;
+
+export const changeAccountState = async (
+  deltaAmount: number,
+  interestRateCalculation = false
+) => {
+  const randomError = Math.random() <= 0.75;
+
+  await delay(2000);
+  return randomError
+    ? (accountBalance = Math.round(accountBalance + deltaAmount))
+    : Promise.reject(
+        interestRateCalculation ? accountBalance : (accountBalance = 0)
+      );
+};
+
+export const actualBalance = () => {
+  return accountBalance;
+};
+
+const delay = async (ms: number) => {
+  return new Promise<void>((resolve) => {
```

Comment:
není `<void>` defaultní hodnota? to tam definuješ IMHO zbytečně

Reply:
no, není 
![image](https://user-images.githubusercontent.com/12272063/169821015-71d3706e-d262-400a-beb9-3309db875226.png)